resources/js/components/Board.js

```diff
@@ -349,7 +349,7 @@ export class Board extends React.Component {
               </center>
             </form>
         </Modal>
-        <form action='/' style={{display: "inline"}}>
+        <form action="javascript:window.history.back();" style={{display: "inline"}}>
```

Comment:
I think the other back button on the settings page also needs this. If you don't want to write the same code over and over, you can just make the back button into a [blade component](https://laravel.com/docs/9.x/blade#components).

Reply:
I believe I did change both the settings and the board back button. Unless you are talking about something else

Reply:
Whoops, didn't even see it! My bad, disregard